components/LanguagePicker.vue

```diff
@@ -0,0 +1,69 @@
+<script lang="ts" setup>
```

Comment:
I'd say that since at least two routes (`login/` and `/`) use this component, let's move to `shared/